### ISSUE TITLE
pythonForUarm Documentation faulty

### ISSUE THREAD

**Issue by lorenz-halt** [uArm-Developer/UArmForPython#3]
Hi,
I tried to follow the instructions on http://developer.ufactory.cc/api/python/ 
But already the first entry points to a non-existing repository https://github.com/uArm-Developer/pyFirmata

How can I proceed? What can I do?
My goal is to use uArmForRos..

Thanks for your help!

**Comment by altjz** [comment]
Hi @ipa-lth , ROS is not compatible with pyuarm yet. We could provide pyFirmata to you. if you want it.

**Comment by lorenz-halt** [comment]
Well, I already did "pip install pyFirmata", although I am not sure if that was a correct move. 

But what do I need to do to get the uArmForRos running? What Arduino program do I need to upload? I am a kind of ROS veteran, so this side is currently not my biggest issue.

**Comment by altjz** [comment]
I submit a commit with [pyFirmata](https://github.com/uArm-Developer/UArmForROS/tree/master/pyFirmata), we made some differences from the official version.

**Comment by lorenz-halt** [comment]
Ok, thx.
Is it correct to upload the uArmFirmata Example onto the Arduino?

**Comment by altjz** [comment]
First, Please use this one Calibrations.ino to Calibrate uArm.
[Calibration Sketch](https://github.com/uArm-Developer/UArmForArduino/tree/75638318cc57f22b57cdb633377853d0fd3b6e15)

Then upload one for Communication. [Firmata EEPROM](https://github.com/uArm-Developer/UArmForROS/tree/master/FirmataEEPROM), Different version with Official Firmata.

**Comment by lorenz-halt** [comment]
Hi Alex,
it had been some time until I found some time to care about the uarm again.
I did a calibration but now I am stuck again with the next steps:

-> Which Firmata ino should I flash onto the uarm?

There is one in a folder called "backups" and one in a folder called "examples". Both do not compile because they are missing "FirmataEEPROM.h".

As already told, I am not a arduino user and don't know much about the arduino part.
